source/technical/reservations.rst

```diff
@@ -156,6 +158,17 @@ Instead of specifying the node type, you may also reserve a specific node by pro
 
    blazar lease-create --physical-reservation min=1,max=1,resource_properties='["=", "$uid", "c9f98cc9-25e9-424e-8a89-002989054ec2"]' --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-custom-lease
 
+.. _disable-blazar-notification:
+.. attention:: 
+   To specify a before_end action, simply add ``before_end=<action_type>`` to ``physical-reservation`` parameter. For example:
+   
+   .. code-block:: bash
+
+      blazar lease-create --physical-reservation min=1,max=1,resource_properties='["=", "$uid", "c9f98cc9-25e9-424e-8a89-002989054ec2"]',before_end=email --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-custom-lease
+      
+   Currenly supported actions include ``default`` [do nothing] and ``email`` [send email notification].
```

Comment:
I think the option `default` should be renamed to `none`, given that the real default is to send an email, and one must explicitly opt-in to the "do nothing" case.

Reply:
I understand that we ended up not implementing this idea?